Example in Readme dont work

```Rust
extern crate aes;
extern crate block_modes;
extern crate hex_literal;

use aes::Aes128;
use block_modes::block_padding::Pkcs7;
use block_modes::{BlockMode, Cbc};
use hex_literal::hex;

fn main() {
    // create an alias for convenience
    type Aes128Cbc = Cbc<Aes128, Pkcs7>;
    let key = hex!("000102030405060708090a0b0c0d0e0f");
    let iv = hex!("f0f1f2f3f4f5f6f7f8f9fafbfcfdfeff");
    let plaintext = b"Hello world!";
    let cipher = Aes128Cbc::new_var(&key, &iv).unwrap();
    // buffer must have enough space for message+padding
    let mut buffer = [0u8; 32];
    // copy message to the buffer
    let pos = plaintext.len();
    buffer[..pos].copy_from_slice(plaintext);
    let ciphertext = cipher.encrypt(&mut buffer, pos).unwrap();
    assert_eq!(ciphertext, hex!("1b7a4c403124ae2fb52bedc534d82fa8"));
    // re-create cipher mode instance and decrypt the message
    let cipher = Aes128Cbc::new_var(&key, &iv).unwrap();
    let mut buf = ciphertext.to_vec();
    let decrypted_ciphertext = cipher.decrypt(&mut buf).unwrap();
    assert_eq!(decrypted_ciphertext, plaintext);
}
```
Errors
```
error[E0599]: no function or associated item named `new_var` found for struct `Cbc<Aes128, Pkcs7>` in the current scope
  --> src/main.rs:16:29
   |
16 |     let cipher = Aes128Cbc::new_var(&key, &iv).unwrap();
   |                             ^^^^^^^ function or associated item not found in `Cbc<Aes128, Pkcs7>`

error[E0599]: no function or associated item named `new_var` found for struct `Cbc<Aes128, Pkcs7>` in the current scope
  --> src/main.rs:25:29
   |
25 |     let cipher = Aes128Cbc::new_var(&key, &iv).unwrap();
   |                             ^^^^^^^ function or associated item not found in `Cbc<Aes128, Pkcs7>`

warning: unused import: `BlockMode`
 --> src/main.rs:7:19
  |
7 | use block_modes::{BlockMode, Cbc};
  |                   ^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

error: aborting due to 2 previous errors; 1 warning emitted

For more information about this error, try `rustc --explain E0599`.
error: could not compile `pet`

To learn more, run the command again with --verbose.
```

--- a/README.md
+++ b/README.md
@@ -63,7 +63,7 @@ type Aes128Cbc = Cbc<Aes128, Pkcs7>;
 let key = hex!("000102030405060708090a0b0c0d0e0f");
 let iv = hex!("f0f1f2f3f4f5f6f7f8f9fafbfcfdfeff");
 let plaintext = b"Hello world!";
-let cipher = Aes128Cbc::new_var(&key, &iv).unwrap();
+let cipher = Aes128Cbc::new_from_slices(&key, &iv).unwrap();
 
 // buffer must have enough space for message+padding
 let mut buffer = [0u8; 32];
@@ -75,7 +75,7 @@ let ciphertext = cipher.encrypt(&mut buffer, pos).unwrap();
 assert_eq!(ciphertext, hex!("1b7a4c403124ae2fb52bedc534d82fa8"));
 
 // re-create cipher mode instance and decrypt the message
-let cipher = Aes128Cbc::new_var(&key, &iv).unwrap();
+let cipher = Aes128Cbc::new_from_slices(&key, &iv).unwrap();
 let mut buf = ciphertext.to_vec();
 let decrypted_ciphertext = cipher.decrypt(&mut buf).unwrap();
 
@@ -86,12 +86,12 @@ With an enabled `std` feature (which is enabled by default) you can use
 `encrypt_vec` and `decrypt_vec` methods:
 
 ```rust
-let cipher = Aes128Cbc::new_var(&key, &iv).unwrap();
+let cipher = Aes128Cbc::new_from_slices(&key, &iv).unwrap();
 let ciphertext = cipher.encrypt_vec(plaintext);
 
 assert_eq!(ciphertext, hex!("1b7a4c403124ae2fb52bedc534d82fa8"));
 
-let cipher = Aes128Cbc::new_var(&key, &iv).unwrap();
+let cipher = Aes128Cbc::new_from_slices(&key, &iv).unwrap();
 let decrypted_ciphertext = cipher.decrypt_vec(&ciphertext).unwrap();
 
 assert_eq!(decrypted_ciphertext, plaintext);

